iib/workers/tasks/build.py

```diff
@@ -792,7 +799,83 @@ def _copy_files_from_image(image, src_path, dest_path):
             log.exception(e)
 
 
-def _adjust_operator_manifests(manifests_path):
+def _add_package_name_suffix(metadata_path, organization=None):
+    """
+    Add the package name suffix if configured for this organization.
+
+    This adds the suffix to the value of
+    ``annotations['operators.operatorframework.io.bundle.package.v1']`` in
+    ``metadata/annotations.yaml``.
+
+    The final package name value is returned as part of the tuple.
+
+    :param str metadata_path: the path to the bundle's metadata directory.
+    :param str organization: the organization this customization is for.
+    :raise IIBError: if the ``metadata/annotations.yaml`` file is in an unexpected format.
+    :return: a tuple with the package name and a dictionary of labels to set on the bundle.
+    :rtype: tuple(str, dict)
+    """
+    annotations_yaml_path = os.path.join(metadata_path, 'annotations.yaml')
+    if not os.path.exists(annotations_yaml_path):
+        raise IIBError('metadata/annotations.yaml does not exist in the bundle')
+
+    with open(annotations_yaml_path, 'r') as f:
+        try:
+            annotations_yaml = yaml.safe_load(f)
+        except yaml.YAMLError:
+            error = 'metadata/annotations/yaml is not valid YAML'
+            log.exception(error)
+            raise IIBError(error)
+
+    package_label = 'operators.operatorframework.io.bundle.package.v1'
```

Comment:
This seems out of place. Should we move it to just above the line where it gets first used?